released/uc_sap_build_apps/step1/terraform.tfvars

```diff
@@ -0,0 +1,27 @@
+#################################
+# Provider configuration
+#################################
+# Your global account subdomain
+globalaccount   = "ticrossa"
+region          = "us10"
+custom_idp      = "alv7kqzip.accounts.ondemand.com"
```

Comment:
Should we remove the custom IDP Id in the sample?

Reply:
Makes sense. Will do.